lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @fileOverview Disallow the use of specific jQuery methods.
+ * @author Marc Lynch
+ */
+'use strict';
+
+/**
+ * Determines if this expression matches the method blacklist invocation.
+ * @param {ASTNode} node The identifier node.
+ * @returns {Boolean} Returns true if the expression matches, otherwise false.
+ */
+
+function get(node, path) {
```

Comment:
Not from astexplorer. I'm following this similar pattern from other rules within this repo. 

I think it provides some value.